File: reactive-demo-backend/src/main/java/dev/divezone/demo/reactive/data/AirlineRepository.java
package dev.divezone.demo.reactive.data;

import java.util.List;

import org.springframework.data.repository.CrudRepository;

import dev.divezone.demo.reactive.model.Airline;

public interface AirlineRepository extends CrudRepository<Airline, Integer> {

    List<Airline> findAirlineByCountry(String country);

}

File: reactive-demo-backend-kotlin/src/main/kotlin/dev/divezone/demo/reactive/data/AirlineRepository.kt
package dev.divezone.demo.reactive.data

import dev.divezone.demo.reactive.model.Airline
import org.springframework.data.repository.CrudRepository

interface AirlineRepository : CrudRepository<Airline, Int> {
    fun findAirlineByCountry(country: String): List<Airline>
    fun save(entity: Airline)
}
